ci/tests/arakoon/extend_promote_demote_test.py

```diff
@@ -309,38 +283,19 @@ def ar_0002_arakoon_cluster_validation_test():
         """
         Arakoon cluster validation
         """
-        storagerouters = GeneralStorageRouter.get_storage_routers()
-        if not len(storagerouters) >= 2:
-            raise SkipTest('Environment has only {0} node(s)'.format(len(storagerouters)))
-
-        pmachines = TestArakoon.get_cluster_pmachines([sr.ip for sr in storagerouters])
-        TestArakoon.validate_arakoon_config_files(pmachines)
-
-    @staticmethod
-    def ovs_3554_4_node_cluster_config_validation_test():
-        """
-        Arakoon config validation of a 4 node cluster
-        """
-        storagerouters = GeneralStorageRouter.get_storage_routers()
-        if not len(storagerouters) >= 4:
-            raise SkipTest('Environment has only {0} node(s)'.format(len(storagerouters)))
-
-        pmachines = TestArakoon.get_cluster_pmachines([sr.ip for sr in storagerouters])
-        TestArakoon.validate_arakoon_config_files(pmachines)
+        TestArakoon.validate_arakoon_config_files(GeneralStorageRouter.get_storage_routers())
 
     @staticmethod
     def ovs_3671_validate_archiving_of_existing_arakoon_data_on_create_test():
         """
         Validate arakoon archiving on extending a cluster with already existing data
         """
-        node_ips = [sr.ip for sr in GeneralStorageRouter.get_storage_routers()]
-        node_ips.sort()
+        first_sr = GeneralStorageRouter.get_storage_routers()[0]
```

Comment:
In the previous version the ips were sorted, however you might not be sure about the sorting here (as you're not sure that get_storage_routers will be stable. I think it might be useful to sort them (on ip) before taking the first one.